boards/others/candlelightfd/candlelightfd.yaml

```diff
@@ -0,0 +1,15 @@
+identifier: candlelightfd
```

Comment:
name file candlelightfd_stm32g0b1xx.yaml and change line to `identifier: candlelightfd/stm32g0b1xx`